### PR TITLE
Xhr load test

### PR DESCRIPTION
**Pull request by kateriley1**
Documentation of updated load test, which now simulates xhr/fetch requests.

### CODE REVIEW

File: spikes/2 - load_testing_xhr.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+## Viewing XHR requests on a webpage
+XHR requests that a dashboard page performs when a filter is changed can be found by:
+1. Loading the dashboard in chrome
+2. Right clicking and selecting inspect and then selecting network
+3. Making a change on the dashboard page in chrome
+4. Selecting request payload
+
+## General usage estimate: 
+500000 monthly users / 30 days average in a month = 1667 daily users <br>
```
Comment (chopkinsmade) [inline spikes/2 - load_testing_xhr.md:23]: can remove this <br> tag, and also the line below

File: spikes/2 - load_testing_xhr.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+Prometheus will be used to gather the CPU Usage and Memory utilisation; the Locust user interface will provide response time and failed user request metrics.
+
+Note: The dashboard has autoscaling configured, with a maximum of 2 instances.
+
+We will break our estimate down into 2 scenarios - General usage and Launch day (expecting a higher usage on launch) and run each scenario load test for 15 minutes.
+
+## Viewing XHR requests on a webpage
+XHR requests that a dashboard page performs when a filter is changed can be found by:
+1. Loading the dashboard in chrome
+2. Right clicking and selecting inspect and then selecting network
```
Comment (chopkinsmade) [inline spikes/2 - load_testing_xhr.md:18]: Only something small, but you can also get this menu by pressing F12

Comment (kateriley1) [reply spikes/2 - load_testing_xhr.md:18]: That is easier, I'll include this shortcut - thank you.

File: spikes/2 - load_testing_xhr.md (hunk 1 above)
Comment (kateriley1) [reply spikes/2 - load_testing_xhr.md:24]: I tried removing the <br> tags, however all the text then appeared on one line.

File: spikes/2 - load_testing_xhr.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+The load test will simulate XHR/Fetch requests on page updates - the application updates part of a page without performing a full page refresh. So when an element on a page of the dashboard is updated, an XHR request is made to perform the update. 
+
+Prometheus will be used to gather the CPU Usage and Memory utilisation; the Locust user interface will provide response time and failed user request metrics.
+
+Note: The dashboard has autoscaling configured, with a maximum of 2 instances.
+
+We will break our estimate down into 2 scenarios - General usage and Launch day (expecting a higher usage on launch) and run each scenario load test for 15 minutes.
+
+## Viewing XHR requests on a webpage
+XHR requests that a dashboard page performs when a filter is changed can be found by:
```
Comment (chopkinsmade) [inline spikes/2 - load_testing_xhr.md:16]: Numbering is off for the last 2 items, if you change them all to 1. then the markdown display will work out the numbering for you

**Review by chopkinsmade — APPROVED** [review]

File: spikes/2 - load_testing_xhr.md (hunk 3 above)
Comment (kateriley1) [reply spikes/2 - load_testing_xhr.md]: Thank you - I've resolved now.

**Review by adrianclay — APPROVED** [review]

**Comment by tom-clarke-dluhc** [comment]
Two very minor comments from me. In spikes/1-basic_load_test.md in the General Usage Estimate section (line 14) there is an extra zero on 50,000 in the equation. Also, in the numbered instructions in 2-load_testing_xhr.md (lines 20-24) I wasn't able to follow these instructions but I think I found the right thing going to the Network tab at the top. Otherwise this is really interesting and thank you for prodding me to look over it more thoroughly!

**Review by tom-clarke-dluhc — APPROVED** [review]

**Pull request merged**